test/spec/mapbox_logo.js

```diff
@@ -0,0 +1,83 @@
+describe('mapbox_logo', function() {
+    var server, element, doc, tileJSON = helpers.tileJSON;
+
+    beforeEach(function() {
+        server = sinon.fakeServer.create();
+        element = document.createElement('div');
+    });
+
+    afterEach(function() {
+        server.restore();
+    });
+
+    it('constructor', function() {
+        var map = L.mapbox.map(element, tileJSON);
+        var mapboxLogoControl = map._mapboxLogoControl;
+        expect(map._mapboxLogoControl instanceof L.Control);
+        expect(map.getTileJSON()).to.eql(helpers.tileJSON);
+    });
+
+    it('is not on tilejson map without mapbox_logo flag', function() {
+        var map = L.mapbox.map(element, tileJSON);
+        var mapboxLogoControl = map._mapboxLogoControl;
+        expect(mapboxLogoControl._mapboxLogo.length).to.eql(0);
+        expect(map.getTileJSON()).to.eql(helpers.tileJSON);
+    });
+
+    it('is on tilejson map with mapbox_logo === true', function() {
+        var map = L.mapbox.map(element, helpers.tileJSON_mapboxlogo);
+        var mapboxLogoControl = map._mapboxLogoControl;
+        expect(mapboxLogoControl._mapboxLogo.length).to.eql(6396);
```

Comment:
Change this to assert `length > 0` or something else that won't start failing if the logo gets visually tweaked.